serverless/transaction-manager/src/utils.rs

```diff
@@ -18,6 +18,7 @@ pub(crate) fn parse_send_error(error: String) -> TxnManagerSendError {
     if error_lowercase.contains("out of gas")
         || (error_lowercase.contains("transaction requires at least")
             && error_lowercase.contains("gas but got"))
+        || error_lowercase.contains("gas too low")
```

Comment:
is this  is the error that you got from sepolia rpc?

Reply:
Yes